inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/kafka/KafkaSourceOperator.java

```diff
@@ -131,8 +127,13 @@ public Map<String, List<StreamSource>> getSourcesMap(InlongGroupInfo groupInfo,
                 if (!Objects.equals(streamId, sourceInfo.getInlongStreamId())) {
                     continue;
                 }
-                if (StringUtils.isEmpty(kafkaSource.getSerializationType()) && StringUtils.isNotEmpty(
-                        sourceInfo.getSerializationType())) {
+
+                if (sourceInfo.getSourceType().equalsIgnoreCase(SourceType.FILE)) {
```

Comment:
Here should including `AUTO_PUSH`?

Reply:
Yes, `AUTO_PUSH` also can customize the data format.